install.sh

```diff
@@ -897,8 +892,11 @@ case $platform in
     info "Amazon platform! Lets get you an RPM..."
     filetype="rpm"
     platform_package="el"
-    # For Amazon Linux 2023 and onwards we can use the 'amazon' packages created instead of 'el' packages
-    if (( $platform_version >= 2 )); then
+    arch="$(uname -p)"
+    # Install amazon packages on AL2 (only aarch64) and 2003 and up (all arch)
```

Comment:
This is not blocking for me but we should update this comment to 2023 in the puppet_agent module at some point.

Reply:
doh! Tim pointed that out and I fixed one place, but not the other. Let me fix that in the module.

Reply:
https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/729

Reply:
Merged your module PR, can you regenerate this PR? Thanks!

Reply:
Updated, thanks for catching it!